plugins/web/opentelemetry-plugin-user-interaction/src/userInteraction.ts

```diff
@@ -52,6 +53,8 @@ export class UserInteractionPlugin extends BasePlugin<unknown> {
     Function,
     Map<string, Map<HTMLElement, Function>>
   >();
+  // for event bubbling
+  private _event2span: WeakMap<Event, Span> = new WeakMap<Event, Span>();
```

Comment:
At first I thought it was a method and I couldn't find it. Can you change the name to  something like for example `_eventsSpanMap` or `_mapOfEventsWithSpan` etc. ?

Reply:
Sure thing; changing to _eventsSpanMap.